src/redux/store.js

```diff
@@ -0,0 +1,10 @@
+import { createStore, applyMiddleware } from 'redux';
+import { createEpicMiddleware } from 'redux-observable';
+import { Reducers } from './reducers';
+import { Epics } from './epics';
+
+const epicMiddleware = createEpicMiddleware();
+
+export default createStore(Reducers.loginLogout, applyMiddleware(epicMiddleware));
```

Comment:
In create store we should use the combine reducer function that is to combine the multiple reducer in a single store..